<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Credit</class>
 <widget class="QDialog" name="Credit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>646</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>646</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>646</width>
    <height>450</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #202020;</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>601</width>
     <height>398</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lb_credit_sum">
        <property name="font">
         <font>
          <family>Arioal</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: white;
	font-size: 13pt;
	font-weight: 600;
	margin-top: 4px;
	margin-bottom: 4px;
}</string>
        </property>
        <property name="text">
         <string>Общая сумма кредита, руб:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ln_credit_sum">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
   	border: 1px solid white;
	color: white;
	font-family: Arial;
	font-size: 14pt;
	font-weight: 600;
	margin-bottom: 4px;
}
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lb_time">
        <property name="styleSheet">
         <string>QLabel {
	color: white;
	font-size: 13pt;
	font-weight: 600;
	margin-top: 4px;
	margin-bottom: 4px;
}</string>
        </property>
        <property name="text">
         <string>Срок:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ln_time">
        <property name="styleSheet">
         <string notr="true">QLineEdit {
   	border: 1px solid white;
	color: white;
	font-family: Arial;
	font-size: 14pt;
	font-weight: 600;
	margin-bottom: 4px;
}</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmb_time">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox{
  border: 1px solid white;
  color: white;
  font-family: Arial;
  font-size: 14pt;
  font-weight: 600;
  margin-bottom: 4px;
}</string>
        </property>
        <item>
         <property name="text">
          <string>лет</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>месяцев</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lb_nterest">
        <property name="font">
         <font>
          <family>Arioal</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: white;
	font-size: 13pt;
	font-weight: 600;
	margin-top: 4px;
	margin-bottom: 4px;
}</string>
        </property>
        <property name="text">
         <string>Процентная ставка, %:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ln_interest">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit {
   	border: 1px solid white;
	color: white;
	font-family: Arial;
	font-size: 14pt;
	font-weight: 600;
	margin-bottom: 4px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="lb_type">
        <property name="font">
         <font>
          <family>Arioal</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: white;
	font-size: 13pt;
	font-weight: 600;
	margin-top: 4px;
	margin-bottom: 4px;
}</string>
        </property>
        <property name="text">
         <string>Тип:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cmb_type">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>14</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox{
  border: 1px solid white;
  color: white;
  font-family: Arial;
  font-size: 14pt;
  font-weight: 600;
  margin-bottom: 4px;
}
</string>
        </property>
        <item>
         <property name="text">
          <string>аннуитетный</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>дифференцированный</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="lb_monthly_payment">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>Arioal</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: white;
	font-size: 13pt;
	font-weight: 600;
	margin-top: 4px;
	margin-bottom: 4px;
}</string>
        </property>
        <property name="text">
         <string>Eжемесячный платеж:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_monthly_payment_res">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: white;
	font-size: 13pt;
	font-weight: 600;
	margin-top: 4px;
	margin-bottom: 4px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="lb_overpayment">
        <property name="font">
         <font>
          <family>Arioal</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: white;
	font-size: 13pt;
	font-weight: 600;
	margin-top: 4px;
	margin-bottom: 4px;
}</string>
        </property>
        <property name="text">
         <string>Переплата по кредиту:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_overpayment_res">
        <property name="font">
         <font>
          <family>Arial</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: white;
	font-family: Arial;
	font-size: 13pt;
	font-weight: 600;
	margin-top: 4px;
	margin-bottom: 4px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="lb_total">
        <property name="font">
         <font>
          <family>Arioal</family>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: white;
	font-size: 13pt;
	font-weight: 600;
	margin-top: 4px;
	margin-bottom: 4px;
}</string>
        </property>
        <property name="text">
         <string>Oбщая выплата:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_total_res">
        <property name="font">
         <font>
          <pointsize>13</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">QLabel {
	color: white;
	font-size: 13pt;
	font-weight: 600;
	margin-top: 4px;
	margin-bottom: 4px;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Fixed</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <widget class="QPushButton" name="pushButton">
        <property name="minimumSize">
         <size>
          <width>150</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
    background-color: #76B9ED;
	color: #202020;
}
QPushButton:hover {
    background-color: rgb(109, 169, 216);
}
QPushButton:pressed {
    background-color: rgb(100, 154, 195);
    color: rgb(64, 98, 124);
}</string>
        </property>
        <property name="text">
         <string>Расчитать</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
